docs/Other SDL Features/Using App Services/index.md

```diff
@@ -220,9 +220,6 @@ unsubscribeServiceData.subscribe = @NO;
 
 ##### Swift
 ```swift
-// Get service data once
-let getServiceData = SDLGetAppServiceData(appServiceType: .media)
-
```

Comment:
Why was this removed?

Reply:
getServiceData is initialized twice in the same code block.